src/styles/main.scss

```diff
@@ -1,7 +1,13 @@
-@import "utils";
-@import "fonts";
-@import "typography";
-
-body {
-  background: $c-gray;
-}
+@import "../styles/utils";
```

Comment:
It seems like you are importing the 'utils' file from an incorrect location. If the file is in the same directory, you should import it as 'utils' not '../styles/utils'.